packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -99,6 +99,65 @@ <h3 id="attachments-multipleFile" hidden>Attachments</h3>
   </div>
   <!-- FileUploader: end -->
 
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Input Multiple with validation</h2>
+
+  <!-- FileUploader: start -->
+  <div id="example_multipleWithAccept" class="FileUploader" data-toggle="fileUploader">
+
+    <!-- List item template: start -->
+    <template data-spirit-snippet="item">
+      <li class="FileUploaderAttachment" data-populate-field="item">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#file" />
+        </svg>
+        <span class="text-truncate" data-populate-field="name"></span>
+        <button type="button" class="FileUploaderAttachment__remove" data-populate-field="button">
+          <span class="accessibility-hidden">Remove</span>
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#close" />
+          </svg>
+        </button>
+      </li>
+    </template>
+    <!-- List item template: end -->
+
+    <!-- FileUploaderInput: start -->
+    <div class="FileUploaderInput" data-spirit-element="wrapper">
+      <label for="fileUploadMultipleWithAccept" class="FileUploaderInput__label">Label</label>
+      <input type="file" id="fileUploadMultipleWithAccept" name="attachment3" class="FileUploaderInput__input" multiple data-spirit-element="input" accept=".png,image/jpeg" />
```

Comment:
```suggestion
      <input type="file" id="fileUploadMultipleWithValidation" name="attachment3" class="FileUploaderInput__input" multiple data-spirit-element="input" accept=".png,image/jpeg" />
```

So it matches the heading?

But it's not important.

Reply:
Is this really necessary? Since it's for internal use and it's tied to the accept attribute, that's why I chose that name. In other examples, for example, we only use numbers, so I won't give this one that much weight.

Reply:
As I said above ⬆️, it's not important. My personal rule is that I derive IDs from headings. It ensures the uniqueness and works better than numbering when we later add more examples between the ones that already exist.

Reply:
Sure, it's just one of those small things 🙂. It doesn't matter now, but we could add it to the guidelines and "fix" it everywhere. I'm putting down a note.

Reply:
I have filed an issue for this.